itest/lnd_misc_test.go

```diff
@@ -230,6 +230,49 @@ func testSphinxReplayPersistence(ht *lntest.HarnessTest) {
 	ht.ForceCloseChannel(carol, chanPoint)
 }
 
+// testFundingExpiryBlocksOnPending checks that after an OpenChannel, and
+// before the funding transaction is confirmed, that the FundingExpiryBlocks
+// field of a PendingChannels decreases.
+func testFundingExpiryBlocksOnPending(ht *lntest.HarnessTest) {
+	// Configure bob to require 5 confs for a channel to be considered open
+	alice, bob := ht.Alice, ht.Bob
+	const numConfs = 5
+	args := []string{
+		fmt.Sprintf("--bitcoin.defaultchanconfs=%v", numConfs),
+	}
+	ht.RestartNodeWithExtraArgs(bob, args)
+
+	ht.ConnectNodes(alice, bob)
+	param := lntest.OpenChannelParams{
+		Amt:     btcutil.Amount(100000),
+		PushAmt: btcutil.Amount(0),
+	}
+	update := ht.OpenChannelAssertPending(alice, bob, param)
+
+	// At this point, the channel's funding transaction will have been
+	// broadcast, but not confirmed. Alice and Bob's nodes should reflect
+	// this when queried via RPC. FundingExpiryBlock should decrease
+	// as blocks are mined, until the channel is confirmed. Let's mine 5
+	// blocks and verify the value of FundingExpiryBlock at each step.
+	for i := 0; i < numConfs; i++ {
```

Comment:
nit: can use `for i := int32(0); i < numConfs; i++ {` to get rid of the two type casts to `int32()` on the line below.